test/Unit/email/subject-test.php

```diff
@@ -1,39 +1,60 @@
 <?php
 declare(strict_types=1);
-
 namespace email;
-
 use jsys\types\email\Subject;
 use PHPUnit\Framework\TestCase;
-
-
+/**
+ * @property  expectExceptionCode
+ */
 class SubjectTest extends TestCase
 {
     public function setUp(): void
     {
         parent::setUp(); // TODO: Change the autogenerated stub
     }
-
-    public function test_must_not_initialize_with_email_subject_greater_then_1024()
+    public function test_must_not_initialize_with_email_subject_greater_then_998()
     {
         $max = Subject::MAX_LENGTH;
-        $actual = str_repeat("a", 1025);
+        $actual = str_repeat("a", 999);
         $this->expectExceptionCode(1004);
-        $this->expectExceptionMessage('Given string ' .$max. ' should not be greater then 1024');
+        $this->expectExceptionMessage('Given string should not be greater then 998');
+        $emailSubject = new Subject($actual);
+    }
+    public function test_must_not_initialize_with_email_subject_each_line_greater_than_78_characters()
+    {
+        $actual = str_repeat("a", 999);
+        echo <<<EOT
+        $actual
+        $actual
+       EOT;
+        Subject::MAX_LENGTH;
+        $this->expectExceptionCode(1004);
+        $this->expectExceptionMessage('Given string should not be greater then 998');
         $emailSubject = new Subject($actual);
     }
-
     public function test_can_initialize_with_valid_subject_characters()
     {
-        $max = Subject::MAX_LENGTH;
-        $actual = str_repeat("a", 1023);
+        $actual = str_repeat("a", 78);
         $emailSubject = new Subject($actual);
         $this->assertInstanceOf('jsys\types\email\Subject', $emailSubject);
         $this->assertEquals('jsys\types\email\Subject', get_class($emailSubject));
         $this->assertEquals($emailSubject->getSubject(), $actual, 'Both must get the same value');
     }
+    public function test_must_not_initialize_with_email_subject_each_line_greater_than_998_characters()
```

Comment:
Function name is not making clear sense.
Better use 'lines' instead of 'each_line'